apps/s_server.c

```diff
@@ -2030,9 +2029,9 @@ static int sv_body(int s, int stype, unsigned char *context)
     unsigned long l;
     SSL *con = NULL;
     BIO *sbio;
-    struct timeval timeout;
+    struct timeval timeout = { 0, 0 };
 #if defined(OPENSSL_SYS_WINDOWS) || defined(OPENSSL_SYS_MSDOS)
-    struct timeval tv;
+    struct timeval tv = { 0, 0 };
```

Comment:
Why are the above changes needed? It doesn't seem to have a lot to do with the rest of the changes in this commit.